src/assets/metadata/aws.json

```diff
@@ -1,311 +1,3661 @@
+        },
+        {
+          "name": "availability_zone_id",
+          "displayName": "Availability zone ID",
+          "description": "The identifier of the Availability Zone in which the file system's One Zone storage classes exist.",
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#availability_zone_id"
+        },
+        {
+          "name": "creation_token",
+          "displayName": "Creation token",
+          "description": "A unique name used as reference when creating the Elastic File System to ensure idempotent file system creation.",
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#creation_token",
+          "rules": {
+            "max": "64",
+            "min": "1"
+          }
+        },
+        {
+          "name": "encrypted",
+          "displayName": "Encrypted",
+          "description": "If true, the disk will be encrypted.",
+          "type": "Boolean",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#encrypted"
+        },
+        {
+          "name": "kms_key_id",
+          "displayName": "Encrypted",
+          "description": "Amazon Resource Name (ARN) for the Key Management Service (KMS) encryption key.",
+          "type": "String",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#kms_key_id"
+        },
+        {
+          "name": "lifecycle_policy",
+          "displayName": "Lifecycle policy",
+          "description": "A file system lifecycle policy",
+          "type": "Object",
+          "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#lifecycle_policy",
+          "attributes": [
+            {
+              "name": "transition_to_ia",
+              "displayName": "Transition to IA",
+              "description": "Indicates how long it takes to transition files to the IA storage class.",
+              "type": "String",
+              "url": "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#transition_to_ia",
+              "rules": {
+                "values": [
+                  "AFTER_1_DAY",
+                  "AFTER_7_DAYS",
+                  "AFTER_14_DAYS",
+                  "AFTER_30_DAYS",
+                  "AFTER_60_DAYS",
+                  "AFTER_90_DAYS"
+                ]
+              }
+            },
+            {
+              "name": "transition_to_primary_storage_class",
+              "displayName": "Transition to primary storage class",
+              "description": "Describes the policy used to transition a file from infequent access storage to primary storage.",
```

Comment:
infequent -> infrequent

Reply:
Official doc is not always right... Done